.github/workflows/ci.yml

```diff
@@ -41,8 +41,8 @@ on:
         required: false
         type: number
         default: 50
-      run_dependency_convergence:
```

Comment:
This breaks compatibility which just creates headaches. It's not too difficult to continue supporting this parameter.

Reply:
Yes I totally agree. As I said in [this comment](https://github.com/apache/flink-connector-shared-utils/pull/23#issuecomment-1775394910) I merged the 2 test parameters because I misunderstood a comment you did on slack. If we put them separated again, no more braking change.